gnuradio-runtime/lib/controlport/thrift/rpcpmtconverters_thrift.cc

```diff
@@ -255,7 +255,8 @@ pmt::pmt_t rpcpmtconverter::to_pmt_f::operator()(const GNURadio::Knob& knob)
     // FIXME: Don't get loggers every time we need to log something.
     gr::logger_ptr logger, debug_logger;
     gr::configure_default_loggers(logger, debug_logger, "rpcpmtconverter");
-    debug_logger->error("ERROR Don't know how to handle Knob Type (from): {}", knob.type);
+    debug_logger->error("ERROR Don't know how to handle Knob Type (from): {}",
+                        to_string(knob.type));
```

Comment:
this uses `std::to_string` to convert the enum to its integer index value and then convert that to a string representation, and then `fmt` embeds that as `std::string` into the log message.

But fmt can print integers just fine, so I think the less surprising code would be:
```suggestion
    debug_logger->error("ERROR Don't know how to handle Knob Type (from): {:d}",
                        fmt::underlying(knob.type));
```

Reply:
(the `:d` is desirable because depending on the platform, or if a developer thinks they know what they're doing while designing `enum`s to be packed into memory, the *underlying type* of the `enum` might be `char`. It's not going to be relevant in this use case, but others will look for where we fixed enum logging, and they will copy it)

Reply:
Thanks, I incorporated your suggestion and updated the commit message.